apps/wallet/src/ui/app/components/menu/content/WalletSettingsMenuList.tsx

```diff
@@ -117,13 +126,19 @@ export function MenuList() {
             icon: <Logout />,
             onClick: () => setIsLogoutDialogOpen(true),
         },
+        {
```

Comment:
last nit, i would maybe place this item before the reset/logout one, so reset is the last element